### PR TITLE
Party menu app

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/party_context_menu.c
Comment (adrienntindall) [inline src/party_context_menu.c:380]: follow function naming conventions

File: src/party_context_menu.c
Comment (adrienntindall) [inline src/party_context_menu.c:902]: dont use snake_case

File: src/party_context_menu.c
Comment (adrienntindall) [inline src/party_context_menu.c:1020]: me when I take my meds

File: src/party_context_menu.c
Comment (adrienntindall) [inline src/party_context_menu.c:1113]: still stay consistent with other function naming styles and keep the first letters capitalized

File: src/party_context_menu.c
Comment (adrienntindall) [inline src/party_context_menu.c:1123]: etc for remaining functions

File: src/party_menu.c
Comment (adrienntindall) [inline src/party_menu.c:1018]: remove decomp comment

File: src/party_menu.c
Comment (adrienntindall) [inline src/party_menu.c:1020]: can this not be removed?

File: src/party_menu.c
Comment (adrienntindall) [inline src/party_menu.c:2510]: SwapHeldItemGiratina?

File: src/party_menu.c
Comment (adrienntindall) [inline src/party_menu.c:2627]: GiveHeldItem?

Comment (adrienntindall) [reply src/party_menu.c:2627]: Also PartyMenuStruct ->PartyMenu

File: src/party_menu.c
Comment (adrienntindall) [inline src/party_menu.c:2643]: Comment what messages read above pls (I prolly missed a ton this file so blanket comment for party_menu.c)

File: src/party_menu.c
Comment (adrienntindall) [inline src/party_menu.c:2672]: what graphics are these? can prolly name the functions since you alr dumped them

File: src/party_menu_items.c
Comment (adrienntindall) [inline src/party_menu_items.c:500]: !!
(just use a comparison lmao)

File: src/party_menu_items.c
Comment (adrienntindall) [inline src/party_menu_items.c:505]: PARTY_SLOT_NONE

File: src/party_menu_list_items.c
Comment (adrienntindall) [inline src/party_menu_list_items.c:356]: try to not use sp terminology once the func is decompiled

File: src/party_menu_list_items.c
Comment (adrienntindall) [inline src/party_menu_list_items.c:800]: Menu action for softboiled?

File: src/party_menu_list_items.c
Comment (adrienntindall) [inline src/party_menu_list_items.c:825]: messages for this file too, just leave them as a comment above

File: src/start_menu.c
Comment (adrienntindall) [inline src/start_menu.c:889]: unk30

File: src/start_menu.c
Comment (adrienntindall) [inline src/start_menu.c:981]: unk_8

File: src/unk_0206B910.c
Comment (adrienntindall) [inline src/unk_0206B910.c:103]: unk2C

File: src/unk_02092BE8.c
Comment (adrienntindall) [inline src/unk_02092BE8.c:57]: unk28

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: src/party_context_menu.c
Comment (PikalaxALT) [reply src/party_context_menu.c:380]: exception to the convention is local functions that don't operate on the file's primary object

File: src/party_context_menu.c
Comment (PikalaxALT) [reply src/party_context_menu.c:902]: this is a valid use case

File: src/party_context_menu.c
Comment (PikalaxALT) [reply src/party_context_menu.c:1113]: i'd rather have this visual distinction for these kinds of helpers

File: src/party_menu.c
Comment (PikalaxALT) [reply src/party_menu.c:1020]: necessary to match. don't ask why.

File: src/party_menu.c
Comment (PikalaxALT) [reply src/party_menu.c:2643]: Most of these are just {PLACEHOLDER} so nothing really interesting here. If people want to look them up, they can cross-reference the GMM. Commenting what the messages say inside the C code creates an opportunity for desync.

File: src/party_menu.c
Comment (PikalaxALT) [reply src/party_menu.c:2672]: no fucking clue lol

File: src/party_menu_items.c
Comment (PikalaxALT) [reply src/party_menu_items.c:505]: not creating a whole constant just for this case

File: src/party_menu_list_items.c
Comment (PikalaxALT) [reply src/party_menu_list_items.c:356]: not a big enough deal to fix, as they'll be renamed once the function is better understood

File: include/overlay_94.h
Comment (adrienntindall) [inline include/overlay_94.h:16]: unkPtr -> partyMenu

File: include/unk_02092BE8.h
Comment (adrienntindall) [inline include/unk_02092BE8.h:32]: can remove comment now

File: src/overlay_94.c
Comment (adrienntindall) [inline src/overlay_94.c:23]: unkPtr -> partyMenu

File: src/party_context_menu.c
Comment (adrienntindall) [inline src/party_context_menu.c:435]: I still think it's a good idea to comment what the messages are but I can do that in a separate PR later if you don't wanna do it rn

File: src/voltorb_flip/voltorb_flip.c
Comment (adrienntindall) [inline src/voltorb_flip/voltorb_flip.c:1973]: unkC -> menuInputStatePtr

**Review by adrienntindall — DISMISSED** [review]
Few very minor things, just go ahead and address em and feel free to merge right after

**Comment by PikalaxALT** [comment]
> Few very minor things, just go ahead and address em and feel free to merge right after

ok but you know that as soon as i push these changes it'll dismiss your review

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**
